src/modules/menstrual-period/menstrual-period.service.ts

```diff
@@ -247,7 +240,7 @@ export class MenstrualPeriodService {
 
         if (startedAtCollection.length === 0 && userProfile[0].initialPeriodDate === null) {
             throw new CustomNotFoundException({
-                code: 'no-initial-period-date',
+                code: 'not-enouch-data',
                 message: 'there is no date',
             });
```

Comment:
Aqui uma correção de digitação no code, seria: 'not-enough-data'. Também precisaria alterar a message pra ficar de acordo com o code. A mensagem poderia ser: 'There is not enough data to forecast the next menstrual periods'.